### PR TITLE
Components/common

### PR DESCRIPTION
**Pull request by fstodulski**

### CODE REVIEW

File: src/file-upload/Fileupload.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { Component } from 'react';
+
+class Fileupload extends Component<any, any> {
```
Comment (stanimirtt) [inline src/file-upload/Fileupload.tsx:4]: FileUpload not Fileupload

Comment (stanimirtt) [inline src/file-upload/Fileupload.tsx:4]: React.Component<void, void>

File: src/components/UserSettingsModal/UserSettingsModa.service.ts [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import { env } from './../../env/environment';
+import { Fetch } from './../../helpers/fetch';
+export function getUserData(id: number): Promise<any> {
+  return Fetch.request(env.securedRoutes + '/user/' + id);
```
Comment (stanimirtt) [inline src/components/UserSettingsModal/UserSettingsModa.service.ts:4]: use template `${....}`

**Review by stanimirtt — APPROVED** [review]

File: src/components/UserSettingsModal/UserSettingsModa.service.ts (hunk 2 above)
Comment (suezzo) [reply src/components/UserSettingsModal/UserSettingsModa.service.ts:4]: File name should be `UserSettingsModal.service.ts`.

**Pull request merged**